lua/net.lua

```diff
@@ -1,10 +1,14 @@
 local gatekeeperc = require("gatekeeperc")
+local ffi = require("ffi")
 
 local M = {}
 
 -- Function that sets up the network.
 function M.setup_block()
 
+	local front_ports = {"0000:83:00.0"}
+	local back_ports = {"0000:85:00.0"}
```

Comment:
I tried using the standard library call for this (`if_nametoindex`), but it returns an error when run with interface names that have been disconnected from the kernel driver. The scripts for binding and unbinding interfaces are also unable to work with interface names when they are not bound to the kernel.

For now, I amended the DPDK binding/unbinding scripts so that whenever they see the name of an interface, they append it (and its PCI address) to the file `lua/gatekeeper_ifs.txt` if it's not already there. You can see this change here: https://github.com/cjdoucette/dpdk/commit/ca9e4d03b720bb958d9c35648cb85117b8c44132

It's not perfect -- for example, if multiple users are using the system and only one executes the bind commands, only she will have the interface names in her repository. So this is the idea but we should think about improvements.